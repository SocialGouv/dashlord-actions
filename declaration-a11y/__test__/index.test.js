const fs = require("fs");
const path = require("path");
const { analyseFile } = require("../index");

const files = fs.readdirSync(path.join(__dirname, "samples"));

files
  .filter((name) => name.endsWith(".html"))
  .forEach((file) => {
    test(file, async () => {
      const result = await analyseFile(path.join(__dirname, "samples", file), {
        url: "https://example.com",
      });
      expect(result).toMatchSnapshot();
    });
  });
