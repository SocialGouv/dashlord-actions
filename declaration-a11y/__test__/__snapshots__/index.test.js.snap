// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`code.html 1`] = `
{
  "declarationUrl": "https://example.com/accessibilite",
  "mention": "Accessibilité : partiellement conforme",
}
`;

exports[`fabrique.html 1`] = `
{
  "declarationUrl": "https://example.com/declaration-accessibilite",
  "mention": "Accessibilité : non conforme",
}
`;

exports[`lba.html 1`] = `
{
  "declarationUrl": null,
  "mention": "Accessibilité : non conforme",
}
`;

exports[`msp.html 1`] = `
{
  "declarationUrl": null,
  "mention": "Accessibilité : non conforme",
}
`;

exports[`opentermsarchive.org.html 1`] = `
{
  "declarationUrl": "https://example.com/fr/accessibility",
  "mention": "Accessibilité : non conforme",
}
`;
